app/controllers/other_validation_change_requests_controller.rb

```diff
@@ -0,0 +1,48 @@
+class OtherValidationChangeRequestsController < ApplicationController
+  before_action :set_planning_application, only: %i[new create show]
+
+  def new
+    @other_validation_change_request = @planning_application.other_validation_change_requests.new
+  end
+
+  def show
+    @other_validation_change_request = @planning_application.other_validation_change_requests.find(params[:id])
+  end
+
+  def create
+    @other_validation_change_request = @planning_application.other_validation_change_requests.new(other_validation_change_request_params)
+    @other_validation_change_request.user = current_user
+    @current_local_authority = current_local_authority
```

Comment:
is this used anywhere?

Reply:
We need the `current_user` but not the `current_local_authority` so I've removed the latter